.github/workflows/custom_build.yml

```diff
@@ -51,37 +51,9 @@ on:
         type: boolean
 
 jobs:
-  matrix:
-    if: github.event.repository.fork == false
```

Comment:
Can you keep this check, because I don't want fork repos run Custom Build

Reply:
Oh, sorry.  I deleted too many codes 🙄